### DISCUSSION TITLE
How to prevent initial window resizing?

### DISCUSSION THREAD

**Discussion by ghost** [rust-windowing/winit#3688] [General]
Hey so I'm updating to winit 0.30.0 and a very annoying issue is that the window **will always resize** on start. This makes it really hard to deal with surfaces. I'm not interested in updating the swapchain yet so I just want to disable resizing entirely and force a window size.

**Comment by daxpedda** [comment]
Sounds like a bug we would need much more information for, like an OS and maybe a minimal example.

Unfortunately this is a bit hard to get now that OPs account doesn't exist anymore.

**Reply by madsmtm** [reply]
I _think_ there are issues with redundant `WindowEvent::Resized` currently on macOS, which I'm working on fixing - but in any case, I think user code should always be prepared to deal with resizing, as they're also triggered e.g. when moving to a monitor with a different scale factor.

> Unfortunately this is a bit hard to get now that OPs account doesn't exist anymore.

Yeah, I've closed the discussion as there isn't really anything actionable for us to do here.
